examples/kubeflow_pipelines_orchestration/config.yaml

```diff
@@ -0,0 +1,18 @@
+name: mnist_pipeline
```

Comment:
for other examples you mentioned in the README the option to run with `-c config`, why not for Kubeflow?